ci.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+./snc.sh
+if [[ $? -ne 0 ]]; then
+  exit 1
+fi
+# wait till the cluster is stable
+sleep 5m
```

Comment:
Not the right PR for mentioning this, if a sleep 5m is needed after `snc.sh` returns, I'd argue it's a snc bug.